src/cascadia/TerminalApp/TerminalPaneContent.h

```diff
@@ -20,7 +22,7 @@ namespace winrt::TerminalApp::implementation
     struct TerminalPaneContent : TerminalPaneContentT<TerminalPaneContent>, BasicPaneEvents
     {
         TerminalPaneContent(const winrt::Microsoft::Terminal::Settings::Model::Profile& profile,
-                            const TerminalApp::TerminalSettingsCache& cache,
+                            const std::shared_ptr<TerminalSettingsCache>& cache,

```

Comment:
we could make this `TerminalSettingsCache*`, honestly, and dispense with the shared_ptr completely. _We never construct a new cache._ Pane content does not outlive the parent Page.